packages/database/src/schema.prisma

```diff
@@ -520,9 +521,9 @@ model AppellantSubmission {
   uploadOriginalApplicationForm     Boolean?
   uploadApplicationDecisionLetter   Boolean?
   uploadAppellantStatement          Boolean?
-  costApplication                   Boolean?
   uploadCostApplication             Boolean?
   uploadChangeOfDescriptionEvidence Boolean?
+  appellantStatement                Boolean?
```

Comment:
Is this needed given we already have uploadAppellantStatement?

Reply:
oh no sorry it's not, good catch